spm_funcs.py

```diff
@@ -55,6 +55,13 @@ def get_spm_globals(fname):
         SPM global metric for each 3D volume in the 4D image.
     """
     # +++your code here+++
+     img = nib.load(fname)
```

Comment:
```suggestion
    img = nib.load(fname)
```

Indentation error in first line - all the lines should have the same indentation.